### PR TITLE
MISUV-5877 - Build - obligations (Foreign property added)

### PR DESCRIPTION
**Pull request by JordHemsworth**

### CODE REVIEW

File: app/controllers/incomeSources/add/CheckForeignPropertyDetailsController.scala [hunk 1]
```diff
@@ -48,7 +48,8 @@ class CheckForeignPropertyDetailsController @Inject()(val authenticate: Authenti
   extends ClientConfirmedController with FeatureSwitching with I18nSupport {
 
   def show(origin: Option[String] = None): Action[AnyContent] = Action {
-    Ok("Check Foreign Property Details - WIP")
+    Redirect(controllers.incomeSources.add.routes.ForeignPropertyAddedObligationsController.show("XAIS00000000005").url)
```
Comment (avelx) [inline app/controllers/incomeSources/add/CheckForeignPropertyDetailsController.scala:51]: I would assume QAs are fine with this part?

File: app/views/incomeSources/add/ForeignPropertyAddedObligations.scala.html [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+
+        @if(sources.quarterlyObligationsDatesYearOne.nonEmpty){
+            <div class="box-simple" id="quarterly">
+                <div class="page_headers govuk-heading-m"> @getMessage("quarterly-heading") </div>
+                <p class="govuk-body"> @getMessage("quarterly-p1") </p>
+
+                <table class="govuk-table">
+                    <thead class="govuk-table__head">
+                        <tr class="govuk-table__row">
+                            <th scope="col" class="govuk-table__header govuk">@(getMessage("table.heading-1")+" "+sources.quarterlyObligationsDatesYearOne.head.inboundCorrespondenceFrom.getYear+" "+getMessage("to")+" "+(sources.quarterlyObligationsDatesYearOne.head.inboundCorrespondenceFrom.getYear + 1))</th>
```
Comment (avelx) [inline app/views/incomeSources/add/ForeignPropertyAddedObligations.scala.html:78]: .getYear returning LocalDate, and we need apply formatting for LocalDate type, like toLongDate;

File: app/views/incomeSources/add/ForeignPropertyAddedObligations.scala.html [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+                                <td class="govuk-table__cell">@(x.inboundCorrespondenceFrom.toLongDate + " " + getMessage("to") + " " + x.inboundCorrespondenceTo.toLongDate)</td>
+                                <td class="govuk-table__cell">@(x.inboundCorrespondenceDue.toLongDate)</td>
+                            </tr>
+                        }
+                    </tbody>
+
+                    @if(sources.quarterlyObligationsDatesYearTwo.nonEmpty){
+                        <thead class="govuk-table__head">
+                            <tr class="govuk-table__row">
+                                <th scope="col" class="govuk-table__header govuk">@(getMessage("table.heading-1")+" "+sources.quarterlyObligationsDatesYearTwo.head.inboundCorrespondenceFrom.getYear+" "+getMessage("to")+" "+(sources.quarterlyObligationsDatesYearTwo.head.inboundCorrespondenceFrom.getYear + 1))</th>
```
Comment (avelx) [inline app/views/incomeSources/add/ForeignPropertyAddedObligations.scala.html:95]: .getYear is returning LocalDate , and we need to apply formatting for this type

File: app/views/incomeSources/add/ForeignPropertyAddedObligations.scala.html (hunk 2 above)
Comment (JordHemsworth) [reply app/views/incomeSources/add/ForeignPropertyAddedObligations.scala.html:78]: as discussed Long date not needed here

File: app/controllers/incomeSources/add/CheckForeignPropertyDetailsController.scala (hunk 1 above)
Comment (JordHemsworth) [reply app/controllers/incomeSources/add/CheckForeignPropertyDetailsController.scala:51]: Will check with QA, perhaps better to wait til Daniel PR merged then merge main on my branch?

Comment (JordHemsworth) [reply app/controllers/incomeSources/add/CheckForeignPropertyDetailsController.scala]: I've removed this duplicate controller now, site displays error however when hitting Ayubs page, before mine, and can't progress through journey

**Review by avelx — APPROVED** [review]
approved

**Pull request merged**